design/ceph/object/store.md

```diff
@@ -351,3 +351,212 @@ spec:
 
 When this feature is enabled the endpoint in OBCs and COSI Bucket Access can be clubbed with bucket name and host name. For OBC  the `BUCKET_NAME` and the `BUCKET_HOST` from the config map combine to `http://$BUCKETNAME.$BUCKETHOST`. For COSI Bucket Access the `bucketName` and the `endpoint` in the `BucketInfo` to `http://bucketName.endpoint`.
+
+### Pool Placement targets and Storage Classes
+Object Storage API allows users to specify where bucket data will be stored during bucket creation. With `<LocationConstraint>` parameter in S3 API and `X-Storage-Policy` header in SWIFT. Similarly, users can override where object data will be stored by setting `X-Amz-Storage-Class` and `X-Object-Storage-Class` during object creation. 
+
+In Ceph these data locations represented by Pools. Placement targets control which Pools are associated with a particular bucket. A bucket’s placement target is selected on creation, and cannot be modified (See [Ceph doc](https://docs.ceph.com/en/latest/radosgw/placement/#pool-placement-and-storage-classes)). Placement target can also define custom Storage classes to override data pool where object will be stored.
+
+Ceph administrator creates set of `index_pool`, `data_pool`, and `data_extra_pool` per each target placement. Select arbitrary names for target placements. For example `hot` and `cold`:
+```yaml
+# Bucket Data pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+# Bucket index pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket data extra pool for hot storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: hot.rgw.buckets.non-ec
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket Data pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.data
+  namespace: rook-ceph 
+spec:
+  ...
+---
+# Bucket index pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.index
+  namespace: rook-ceph
+spec:
+  ...
+---
+# Bucket data extra pool for cold storage
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: cold.rgw.buckets.non-ec
+  namespace: rook-ceph 
+spec:
+  ...
+```
+If needed, create additional pool to use it in custom StorageClass, for example `GLACIER`:
+```yaml
```

Comment:
This is a nit, but to help with visual separation during review, could you add empty lines before and after code block marks? I've had to reread a few times catch that this ended a block and a new block started after the text.

Reply:
More than a nit, I think Sphinx enforces that.

Reply:
fixed